internal/tests/cluster/session_cleanup_test.go

```diff
@@ -0,0 +1,509 @@
+package cluster
```

Comment:
Overall (maybe nit) on the file: Convention for us is not to pass require objects around but rather testing.T objects. Partly it's due to Go convention, partly because the only option with require is to fail immediately -- it precludes you from deciding you want to assert without failing, and partly it's because you can't call `t.Log` and friends since you don't have access to the original testing.T.